finality-aleph/src/aggregation/mod.rs

```diff
@@ -41,23 +41,16 @@ pub type LegacyAggregator<'a, N> = legacy_aleph_aggregator::IO<
     NoopMetrics,
 >;
 
-pub type CurrentSignableBlockHash = current_aleph_aggregator::SignableHash<BlockHash>;
-pub type CurrentRmc<'a> =
-    current_aleph_bft_rmc::ReliableMulticast<'a, CurrentSignableBlockHash, Keychain>;
-pub type CurrentAggregator<'a, N> = current_aleph_aggregator::IO<
-    BlockHash,
-    CurrentRmcNetworkData,
-    NetworkWrapper<CurrentRmcNetworkData, N>,
-    SignatureSet<Signature>,
-    CurrentRmc<'a>,
->;
+pub type CurrentAggregator<N> =
+    current_aleph_aggregator::IO<BlockHash, NetworkWrapper<CurrentRmcNetworkData, N>, Keychain>;
 
+#[allow(clippy::large_enum_variant)]
```

Comment:
Is Box usage here cumbersome?

Reply:
I guess this is also something we can ignore as this enum is not used that mich

Reply:
It's rather surprizing to me that this showed up now – is it because something now uses a cloned keychain rather than a reference or something like that?

Reply:
Current aggregator directly uses the rmc service instead a `Multicast` trait. Maybe this is the reason. I can wrap around box